### DISCUSSION TITLE
en/posts/tips/multi-environments-project-configuration/

### DISCUSSION THREAD

**Discussion by giscus** [Kapusch/blog-dotnet-maui#13] [Blog comments]
# en/posts/tips/multi-environments-project-configuration/

To validate new features in your application, it is best to test them in an isolated environment. This will prevent any risk of data loss from your users. In this document, you will learn how to set up different environments for your mobile application.

https://www.mauicestclair.fr/en/posts/tips/multi-environments-project-configuration/

**Comment by cjimenezCT** [comment]
Images aren't loading for me on this page

**Reply by Kapusch** [reply]
Thanks for informing!
Images are missing for all English pages, I will remedy to this.

**Reply by Kapusch** [reply]
Hi @cjimenezCT , I discovered that there is a breaking change with the latest versions of Hugo library, I had to downgrade it temporarily.
Can you please try again and see if it works? (You should refresh the page without cache)
Thanks again for your comment!

**Reply by cjimenezCT** [reply]
They're working now
